tiledb/sm/consolidator/fragment_consolidator.cc

```diff
@@ -841,7 +841,7 @@ void FragmentConsolidator::set_query_buffers(
   const auto& array_schema = query->array_schema();
   auto dim_num = array_schema.dim_num();
   auto dense = array_schema.dense();
-  auto attributes = array_schema.attributes();
+  auto& attributes = array_schema.attributes();
```

Comment:
I'm seeing a few of these, was there a compiler error that I'm not thinking of, or did you add these realizing it avoids the copy?

Reply:
Compiler error!

Reply:
Thanks!